src/rez/package_cache.py

```diff
@@ -863,3 +957,13 @@ def _get_hash_path(self, variant):
         dirs.append(hash_dirname)
 
         return os.path.join(*dirs)
+
+    @staticmethod
+    def _print_with_spinner(message, ticks):
```

Comment:
Can we use `rez.vendor.progress.spinner` instead? We also have a small wrapper in rez.util.ProgressBar` that we use in rez-search (https://github.com/AcademySoftwareFoundation/rez/blob/main/src/rez/package_search.py#L69).

Reply:
Yep, updated.
It's not really a progress bar since it's just waiting for the status to change.  I suppose we could monitor the state of each copying package, but I think a spinner is generally okay for this.